### PR TITLE
Rename Traces.Subscriber --> Traces.TraceObserver, Traces.Headers -->Traces.HttpHeaders

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java [hunk 1]
```diff
@@ -24,18 +24,29 @@
...
-    public interface Headers {
+    /** Zipkin-compatible HTTP header names. */
+    public interface HttpHeaders {
         String TRACE_ID = "X-B3-TraceId";
         String PARENT_SPAN_ID = "X-B3-ParentSpanId";
         String SPAN_ID = "X-B3-SpanId";
     }
 
+    /**
+     * Represents the event receiver for trace completion events. Implementations are invoked synchronously on the
```
Comment (markelliot) [inline tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java]: trace completion -> span completion?

File: tracing/src/main/java/com/palantir/remoting1/tracing/AsyncSpanObserver.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting1.tracing;
+
+/**
+ * Created by rfink on 10/12/16.
```
Comment (markelliot) [inline tracing/src/main/java/com/palantir/remoting1/tracing/AsyncSpanObserver.java]: not the style of the codebase?

File: tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java [hunk 3]
```diff
@@ -45,7 +56,7 @@
     };
 
     // Thread-safe set implementation
-    private static final Set<Subscriber> SUBSCRIBERS = Sets.newConcurrentHashSet();
+    private static final Set<SpanObserver> observers = Sets.newConcurrentHashSet();
```
Comment (markelliot) [inline tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java:59]: nit: we tend to make constants (event private ones) all caps, but w/e

Comment (uschi2000) [reply tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java:59]: well, this isn't a CONSTANT, it's a static final thing that changes over time, similar to `log`

File: tracing/src/main/java/com/palantir/remoting1/tracing/AsyncSpanObserver.java (hunk 2 above)
Comment (uschi2000) [reply tracing/src/main/java/com/palantir/remoting1/tracing/AsyncSpanObserver.java]: didn't mean to commit this file, reverting

File: tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java (hunk 1 above)
Comment (uschi2000) [reply tracing/src/main/java/com/palantir/remoting1/tracing/Traces.java]: done

**Review by markelliot — APPROVED** [review]

**Comment by uschi2000** [comment]
wow, 12mins is long

**Comment by markelliot** [comment]
yeah, I think part of the problem is that our tests start and stop webservers constantly.

**Comment by uschi2000** [comment]
yep

**Pull request merged**